GCR/composite.py

```diff
@@ -17,22 +17,27 @@
 MATCHING_FORMAT = None
 MATCHING_ORDER = tuple()
 
-
-def _match(index_this, index_main, sorter=None):
-    if sorter is None:
+def _match(index_this, index_main, sorter=None, secondary=None, sort_descending=True):
```

Comment:
```suggestion
def _match(index_this, index_main, sorter=None, secondary=None, sort_descending=False):
```
Default to False to be consistent with most sort functions